### PR TITLE
add page for project mgmt and calendaring

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

**Comment by stefaniebutland** [comment]
addresses https://github.com/NASA-Openscapes/earthdata-cloud-cookbook/issues/279

**Comment by jules32** [comment]
Thanks Stef, this is excellent. I've put a few comments here and we can merge with Bri on Tuesday. Thanks!

**Comment by jules32** [comment]
- [ ] Also to add: screenshot of Calendar/Roadmap view

File: contributing/calendar-project-management.qmd [hunk 1]
```diff
@@ -0,0 +1,46 @@
+## GitHub for Project Management and Calendaring
+
+People want to be able to see ongoing NASA Openscapes projects, their status, and where they fit in our calendars. The way we use [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) for this purpose will evolve as people see what works for them.
+
```
Comment (jules32) [inline contributing/calendar-project-management.qmd:4]: I'd suggest putting the screenshot up here: (show not tell, early on) - with the new screenshot from today's This Week / This Quarter / This Year

File: contributing/calendar-project-management.qmd [hunk 2]
```diff
@@ -0,0 +1,46 @@
+## GitHub for Project Management and Calendaring
+
+People want to be able to see ongoing NASA Openscapes projects, their status, and where they fit in our calendars. The way we use [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) for this purpose will evolve as people see what works for them.
```
Comment (jules32) [inline contributing/calendar-project-management.qmd:3]:
```suggestion
People want to be able to see ongoing NASA Openscapes and DAAC events, their status, and where they fit in our calendars. The way we use [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) for this purpose will evolve as people see what works for them.
```

**Comment by stefaniebutland** [comment]
NASA Openscapes and DAAC projects and events

File: contributing/calendar-project-management.qmd [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Here we'll use the [workshop-planning](https://github.com/NASA-Openscapes/workshop-planning) repo as an example. The way we use it will evolve to meet people's needs and use cases.
+
+### Issues
+
+Are you leading a workshop? Open an issue in the workshop-planning repo. The issue title should have the workshop name and date. Add the issue to the MainPlanning project. We can do this either from the issue or from the project.
+
+### Labels
+
+Add one or more labels to the issue, like `workshop`, or the names of the participating DAACs.
```
Comment (stefaniebutland) [inline contributing/calendar-project-management.qmd:28]:
```suggestion
Add one or more labels to the issue, like `workshop`, or the names of the participating DAACs. Test suggested commit
```

File: contributing/calendar-project-management.qmd [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+A GitHub project can only include repos that are in the same GitHub organization. [NASA-Openscapes/Projects/MainPlanning](https://github.com/orgs/NASA-Openscapes/projects/7) cannot track issues from [nsidc/earthaccess](https://github.com/nsidc/earthaccess), but we can add high level issues (to the how-we-work repo?) to see `earthaccess` work on our MainPlanning radar.
+:::
+
+## How are we using it?
+
+Here we'll use the [workshop-planning](https://github.com/NASA-Openscapes/workshop-planning) repo as an example. The way we use it will evolve to meet people's needs and use cases.
+
+### Issues
+
+Are you leading a workshop? Open an issue in the workshop-planning repo. The issue title should have the workshop name and date. Add the issue to the MainPlanning project. We can do this either from the issue or from the project.
```
Comment (BriannaLind) [inline contributing/calendar-project-management.qmd:24]:
```suggestion
Are you leading a workshop or attending a meeting? Open an issue in the workshop-planning repo. The issue title should have the workshop name and date. Add the issue to the MainPlanning project. We can do this either from the issue or from the project.
```

**Review by BriannaLind — COMMENTED** [review]
Made a minor change.

File: contributing/calendar-project-management.qmd [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+People want to be able to see ongoing NASA Openscapes projects, their status, and where they fit in our calendars. The way we use [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) for this purpose will evolve as people see what works for them.
+
+**The [MainPlanning](https://github.com/orgs/NASA-Openscapes/projects/7) Project** pulls from several repositories in the NASA-Openscapes GitHub organization:
+
+-   [how_we_work](https://github.com/NASA-Openscapes/how_we_work)
+-   [workshop-planning](https://github.com/NASA-Openscapes/workshop-planning) for planning & calendaring workshops
+-   [earthdata-cloud-cookbook](https://github.com/NASA-Openscapes/earthdata-cloud-cookbook)
+-   cloud-infrastructure (to be created)
+    -   MATLAB
+    -   2i2c access policy & fledging (TODO: [2i2cAccessPolicies](https://github.com/NASA-Openscapes/2i2cAccessPolicies) is currently a repo - should this move into Cookbook?)
```
[the remark this answers is not in the document]
Comment (stefaniebutland) [reply contributing/calendar-project-management.qmd:12]: thanks @jules32

File: contributing/calendar-project-management.qmd [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+-   [how_we_work](https://github.com/NASA-Openscapes/how_we_work) - for tracking `earthaccess` and other items not included in specific repos listed next
+-   [workshop-planning](https://github.com/NASA-Openscapes/workshop-planning) for planning & calendaring workshops
+-   [earthdata-cloud-cookbook](https://github.com/NASA-Openscapes/earthdata-cloud-cookbook)
+-   cloud-infrastructure (to be created); currently conversations tracked in [2i2cAccessPolicies](https://github.com/NASA-Openscapes/2i2cAccessPolicies)
+    -   MATLAB
+    -   2i2c access policy & fledging (TODO: [2i2cAccessPolicies](https://github.com/NASA-Openscapes/2i2cAccessPolicies) is currently a repo - should this move into Cookbook?)
+
+::: callout-note
+A GitHub project can only include repos that are in the same GitHub organization. [NASA-Openscapes/Projects/MainPlanning](https://github.com/orgs/NASA-Openscapes/projects/7) cannot track issues from [nsidc/earthaccess](https://github.com/nsidc/earthaccess), but we can add high level issues (to the how-we-work repo?) to see `earthaccess` work on our MainPlanning radar.
```
[the remark this answers is not in the document]
Comment (stefaniebutland) [reply contributing/calendar-project-management.qmd:15]: We love this kind of drive-by. Thank you @jhkennedy you're right! Will delete this note

File: contributing/calendar-project-management.qmd [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+**The [MainPlanning](https://github.com/orgs/NASA-Openscapes/projects/7) Project** pulls from several repositories in the NASA-Openscapes GitHub organization:
+
+-   [how_we_work](https://github.com/NASA-Openscapes/how_we_work) - for tracking `earthaccess` and other items not included in specific repos listed next
+-   [workshop-planning](https://github.com/NASA-Openscapes/workshop-planning) for planning & calendaring workshops
+-   [earthdata-cloud-cookbook](https://github.com/NASA-Openscapes/earthdata-cloud-cookbook)
+-   cloud-infrastructure (to be created); currently conversations tracked in [2i2cAccessPolicies](https://github.com/NASA-Openscapes/2i2cAccessPolicies)
+    -   MATLAB
+    -   2i2c access policy & fledging (TODO: [2i2cAccessPolicies](https://github.com/NASA-Openscapes/2i2cAccessPolicies) is currently a repo - should this move into Cookbook?)
+
+::: callout-note
```
Comment (stefaniebutland) [inline contributing/calendar-project-management.qmd:14]:
```suggestion
```

File: contributing/calendar-project-management.qmd (hunk 6 above)
Comment (stefaniebutland) [inline contributing/calendar-project-management.qmd:15]:
```suggestion
```

File: contributing/calendar-project-management.qmd [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+A GitHub project can only include repos that are in the same GitHub organization. [NASA-Openscapes/Projects/MainPlanning](https://github.com/orgs/NASA-Openscapes/projects/7) cannot track issues from [nsidc/earthaccess](https://github.com/nsidc/earthaccess), but we can add high level issues (to the how-we-work repo?) to see `earthaccess` work on our MainPlanning radar.
+:::
```
Comment (stefaniebutland) [inline contributing/calendar-project-management.qmd:16]:
```suggestion
```

File: contributing/calendar-project-management.qmd [hunk 9]
```diff
@@ -0,0 +1,46 @@
+## GitHub for Project Management and Calendaring
+
+People want to be able to see ongoing NASA Openscapes and DAAC events, their status, and where they fit in our calendars. The way we use [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) for this purpose will evolve as people see what works for them.
```
Comment (stefaniebutland) [inline contributing/calendar-project-management.qmd:3]:
```suggestion
People want to be able to see ongoing NASA Openscapes and DAAC events, their status, and where they fit in our calendars. The way we use [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) for this purpose will evolve as people see what works for them.

![MainPlanning GitHub Project - Status Table view](images/github-project-views.png){width="85%"}
```

File: contributing/calendar-project-management.qmd [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+Labels are defined at the repo level. If you want to add a new label, check what already exists in other repos and add a new label by copying label names and color codes. (TODO: customize NASA-Openscapes org default labels - [how to](https://docs.github.com/en/organizations/managing-organization-settings/managing-default-labels-for-repositories-in-your-organization))
+:::
+
+### Viewing a Project
+
+We've set up two ways to view the MainPlanning Project: as a Status Table (<https://github.com/orgs/NASA-Openscapes/projects/7/views/1>) or a Roadmap Schedule (<https://github.com/orgs/NASA-Openscapes/projects/7/views/3>).
+
+Notice the URLs above. Each links to a specific view of the same Project. Anyone can add a new view, sort the issues to suit their needs, name it, and save it. It will have its own URL.
+
+![MainPlanning GitHub Project - Status Table view](images/github-project-views.png){width="85%"}
```
Comment (stefaniebutland) [inline contributing/calendar-project-management.qmd:42]:
```suggestion
```

**Review by stefaniebutland — COMMENTED** [review]
yay

**Review by BriannaLind — APPROVED** [review]
Made a minor change.

File: _quarto.yml [hunk 11]
```diff
@@ -146,6 +146,8 @@ website:
       - section: "Contributing"
         href: contributing/index.qmd
         contents:
+          - text: "Project Management and Calendaring"
```
Comment (stefaniebutland) [inline _quarto.yml:149]:
```suggestion
          - text: "Calendaring and Project Management"
```

**Pull request merged**